### PR TITLE
Ci-383 update api to use elk5

### PR DESCRIPTION
**Pull request by mapaux**

### CODE REVIEW

File: src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/ElasticsearchClientFactory.cs [hunk 1]
```diff
@@ -16,16 +19,31 @@ public ElasticsearchClientFactory(IConfigurationSettings applicationSettings)
 
         public IElasticClient Create()
         {
-            using (var settings = new ConnectionSettings(new StaticConnectionPool(_applicationSettings.ElasticServerUrls)))
+            ConnectionSettings settings;
```
Comment (MarkFCain) [inline src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/ElasticsearchClientFactory.cs]: I don't think this IDisposable ConnectionSettings object has an explicit Dispose.

File: src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/FrameworkRepository.cs [hunk 2]
```diff
@@ -107,5 +103,27 @@ public FrameworkCodeSummary GetFrameworkByCode(int frameworkCode)
 
             return document != null ? _frameworkMapping.MapToFrameworkCodeSummary(document) : null;
         }
+
```
Comment (davehowlett) [inline src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/FrameworkRepository.cs]: Is it possible to refactor the code blocks like this to be generic?

`new SearchDescriptor<FrameworkSearchResultsItem>()
 +                .Index(_applicationSettings.ApprenticeshipIndexAlias)
 +                .Type(Types.Parse("frameworkdocument"))
 +                .From(0)
 +                .Sort(sort => sort.Ascending(f => f.FrameworkId))
 +                .Take(take)
 +                .MatchAll()` 

This pattern is in a lot of classes and they seem to look very similar

File: src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/FrameworkRepository.cs [hunk 3]
```diff
@@ -107,5 +103,27 @@ public FrameworkCodeSummary GetFrameworkByCode(int frameworkCode)
...
+        private ISearchRequest GetAllFrameworksSearchDescriptor(int take)
```
Comment (davehowlett) [inline src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/FrameworkRepository.cs]: can these common searches be refactored?

File: src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/StandardRepository.cs [hunk 4]
```diff
@@ -69,5 +66,27 @@ public Standard GetStandardById(string id)
 
             return document != null ? _standardMapping.MapToStandard(document) : null;
         }
+
```
Comment (davehowlett) [inline src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/StandardRepository.cs]: can these common searches be refactored?

**Pull request merged**